pytest_splunk_addon/standard_lib/data_models/Change.json

```diff
@@ -61,13 +61,13 @@
         {
           "name": "object_path",
           "validity": "if(like(object_path,'%\\\"%'),null(),object_path)",
-          "type": "required",
+          "type": "optional",
           "comment": "The path of the modified resource object, if applicable (such as a file, directory, or volume)."
         },
         {
           "name": "result",
-          "type": "optional",
-          "expected_values": ["lockout"],
+          "condition": "status=failure",
```

Comment:
this is supposed to be the literal string the vendor uses in the log so there no constraint on what may be present.

Reply:
So we mark it as optional? @justin-splunk 
or keep it required but not match the value?

Reply:
The `result` provides the reason of the `action`'s failure, so I suggest:
-  if the `status`=failure, then the `result` is required (yes it may not always be present in the raw, but it will prompt the SME to investigate why the `action` failed - directly from the raw or indirectly from the vendor docs). 
- if the `status`=success, the `result` is optional.

Reply:
@alexeisuv 
So current condition is that if status=failure then result should be required, else it is optional